### PR TITLE
fix(sequelize.col): associations

### PR DESCRIPTION
**Pull request by warkenji**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### Description of change

Sequelize.col doesn't support use of associations

### CODE REVIEW

**Comment by warkenji** [comment]
Sorry for the previous test

**Comment by warkenji** [comment]
It's just by chance and I thought this change was logical compared to the associations between tables, that's all ^^"

File: test/integration/include.test.js [hunk 1]
```diff
@@ -48,6 +48,53 @@ describe(Support.getTestDialectTeaser('Include'), () => {
...
+              Table2Id: table2.get('id'),
+              value: 7
+            }
+          ], {
+            validate: true
+          });
+        });
+      }).then(() => {
+        return Table1.findAll({
+          raw: true,
```
Comment (sushantdhiman) [inline test/integration/include.test.js]: Remove raw and assert on return value

**Pull request merged**